### PR TITLE
Update UbuntuSetup.sh

### PR DESCRIPTION
**Pull request by mkuelker**
Needed to fixe the script a little

### CODE REVIEW

**Review by vnajack — APPROVED** [review]
Excellent, thanks for updating this! It gives us steps for installing on Linux, so we can add to our README knowing these steps work! Awesome!

File: docs/research/API_load_test/UbuntuSetup.sh [hunk 1]
```diff
@@ -6,30 +6,40 @@
...
+cd speaking-portal-project/
 sudo apt update
 sudo apt-get install unzip
 curl -LO https://github.com/DanielSWolf/rhubarb-lip-sync/releases/download/v1.13.0/Rhubarb-Lip-Sync-1.13.0-Linux.zip
 unzip Rhubarb-Lip-Sync-1.13.0-Linux.zip
 mv Rhubarb-Lip-Sync-1.13.0-Linux rhubarb
 rm Rhubarb-Lip-Sync-1.13.0-Linux.zip
 echo "Install app dependencies"
-npm install --production
+npm install
```
Comment (LoganParker) [inline docs/research/API_load_test/UbuntuSetup.sh:40]: We might need to update the production packages, but this should be switched back once we do!

**Review by LoganParker — APPROVED** [review]
Thank you for getting this all set up matt!! Super awesome you managed to get it working :)

File: docs/research/API_load_test/UbuntuSetup.sh [hunk 2]
```diff
@@ -6,30 +6,40 @@
...
 node --version
+echo "Installing Typescript"
+sudo npm install -g typescript
 echo "Installing ffmpeg"
 sudo apt update && sudo apt upgrade
 sudo apt install ffmpeg
 ffmpeg -version
 echo "Installing Rhubarb"
+cd The-Speaking-Portal-Project/
+cd speaking-portal-project/
```
Comment (LoganParker) [inline docs/research/API_load_test/UbuntuSetup.sh]:
```suggestion
cd The-Speaking-Portal-Project/speaking-portal-project/
```
This should work just fine?

Comment (mkuelker) [reply docs/research/API_load_test/UbuntuSetup.sh]: Yeah this is a better way of doing things

**Comment by vnajack** [comment]
- related to #238 
- related to #240

**Pull request merged**